features/FEATURE_BLE/ble/SecurityManager.h

```diff
@@ -735,9 +735,23 @@ class SecurityManager {
     // MITM
     //
 
+    /**
+     * Generate OOB data with the given address. If Secure Connections is supported this will
+     * also generate Secure Connections OOB data on top of legacy pairing OOB data. This can be used
+     * to generate such data before any connections take place.
+     *
+     * @param[in] address The local address you will use in the connection using this OOB data.
+     * @return BLE_ERROR_NONE or appropriate error code indicating the failure reason.
+     */
+    virtual ble_error_t generateOOB(const ble::address_t *address) {
```

Comment:
What happen if I don't use the address passed in parameter but another one ? 
What if the user don't know the address that will be used ? 

Could you add comments for these use cases ?

Reply:
If feed the API incorrect data (like an invalid address) then the behaviour is undefined.
Will add comments.